boards/xtensa/intel_adsp_baytrail/doc/index.rst

```diff
@@ -0,0 +1,104 @@
+.. _Up_Squared_Audio_DSP:
+
```

Comment:
looks like this documentation is being duplicated across all new boards and might be a copy of the already exsiting UP board docs: boards/xtensa/up_squared_adsp/doc/index.rst

Remove all the duplicated documents and maybe add a paragraph for each with a link to some common docs (for example: the exiting up sqaured board)

Reply:
I guess this is one of the reasons documentation is failing...